inst/www/shared/selectize/scss/selectize.bootstrap5.scss

```diff
@@ -1,3 +1,200 @@
-$input-line-height-sm: $form-select-line-height !default;
-@import 'selectize.bootstrap4';
-.selectize-control{padding:0;}
+
+$enable-shadows: true !default;
+$select-font-family: inherit !default;
+$select-font-size: inherit !default;
+$select-line-height: $input-btn-line-height !default; //formerly line-height-computed
```

Comment:
Just a note that we should use `$input-btn-*` for `font-family` and `font-size` as well. See https://github.com/rstudio/bslib/pull/763